index.html

```diff
@@ -30,7 +30,7 @@ <h2>日本で近日開催のイベント</h2>
   <!-- ↓直近で開催予定のイベントページがまだない場合 -->
 
   <div class="span12" style="padding-bottom: 2em;">
-    coming soon..
+    <a href="https://railsgirls.com/nagasaki_2nd.html">Rails Girls Nagasaki 2nd</a>
```

Comment:
ここは、events.html と同じように、ロゴ画像付きで表示させて欲しいです。

```suggestion
    <a href="https://railsgirls.com/nagasaki_2nd.html" class="span4 event" style="background:url(/images/railsgirls-sq.png) center -20px  / 60% no-repeat;">
      <h3>Rails Girls Nagasaki 2nd<small>2025/02/14〜2025/02/15</small></h3>
    </a>                                                                                                                                          
```